src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -79,6 +79,22 @@ module CommandResponse =
       GlyphChar: string
     }
 
+  type ResponseError<'T> =
+    {
+      Code: int
+      Message: string
+      Data: 'T
```

Comment:
I would think about calling this field `AdditionalData` or `Info`.  Otherwise it will be confusing to use - `resp.Data.Data.Field`